cmd/createcluster.go

```diff
@@ -265,13 +328,34 @@ func getKeys(conf clusterConfig) ([]*bls_sig.SecretKey, error) {
 	}
 
 	// TODO(corver): Add flag to generate more distributed-validators than 1
+	var secrets []*bls_sig.SecretKey
+	for i := 0; i < conf.NumDVs+1; i++ { // Note that default value of conf.NumDVs is 0
+		_, secret, err := tbls.KeygenWithSeed(rand.Reader)
+		if err != nil {
+			return nil, err
+		}
+
+		secrets = append(secrets, secret)
+	}
+
+	return secrets, nil
+}
+
+// writeDepositData writes deposit data to disk for the DVs in a cluster.
+func writeDepositData(config clusterConfig, b [][]byte) error {
+	depositPath := path.Join(config.ClusterDir, "deposit-data.json")
+
+	bytes, err := deposit.MarshalDepositDatas(b)
```

Comment:
mmmm, this isn't correct, bytes is already marshalled...?

Reply:
the method signature is changed in https://github.com/ObolNetwork/charon/pull/557